Assets/Scenes/LevelOneScene.unity

```diff
@@ -747,6 +747,7 @@ MonoBehaviour:
   platforms: []
```

Comment:
Since you shortened the platform width, you should also shrink the checkpoint model size. As of now, the checkpoint is hanging off the edge a little bit: 
![image](https://user-images.githubusercontent.com/45300621/203191017-21655ad1-d712-41f3-a3ee-2aa7feff21a0.png)